spec/graphql/directive_spec.rb

```diff
@@ -8,6 +8,7 @@
         # plain fields:
         skipFlavor: flavor @skip(if: true)
         dontSkipFlavor: flavor @skip(if: false)
+        dontSkipDontIncludeFlavor: flavor @skip(if: false), @include(if: false)
```

Comment:
Yes it does mean *don't* skip and *don't* include.

The thing is that having `@skip(if: true) @include(if: true)` would result in skipping (since skip has precedence) and I'd rather test the `presence` of the field instead of testing the absence of the field, which could be anything.

No strong opinion on this though, let me know what you think.

In that test, if skip